### ISSUE TITLE
Fonts used in Easy Translator, and warning text

### ISSUE THREAD

**Issue by katzenbe** [MscrmTools/MsCrmTools.Translator#14]
What fonts are  used on the "Export Translations" and "Import Translations" tabs of the Easy Translator?  Also, can I suggest rewording the warning text like this:

Prior to using this tool ensure that you have a backup of the current customizations
If there is a problem the backup can be re-imported to restore original translations

The Excel file used to import translations must meet the following conditions:
•	Worksheet names have not been changed
•	Cells with colored backgrounds have not been changed
•	No rows have been added to any worksheet in the file
•	No worksheets have been added to the file
•	The only added columns are for additional languages

**Comment by MscrmTools** [comment]
What about fonts? Is there a display problem?

Thanks for the rewording, I will update the text (sorry for my bad englsih)

**Comment by katzenbe** [comment]
The font looks great.. I want to use the same font in an application but can't quite tell what it is. Calibri maybe, or Segoe UI?

**Comment by MscrmTools** [comment]
It's Segoe UI
